app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts
```

Comment:
- [ ] In the last review I also suggested reviewing the recent_posts method because it should show only three posts in user#show page, but it is showing all of them. Please 🙏🏽 fix this error by replacing posts with the correct method in this line 👇🏽 

`@posts = @user.recent_posts`